client/src/components/ui/Button.svelte

```diff
@@ -0,0 +1,36 @@
+<script lang="ts">
+  import { ButtonType } from "../types.ts";
```

Comment:
Let's use single quotes for consistency here.
```suggestion
  import { ButtonType } from '../types.ts';
```